pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -107,6 +110,16 @@ func status() registry.State {
 			glog.Warningf("Warning ! minimum required version for podman is %s. your version is %q. minikube might not work. use at your own risk. To install latest version please see https://podman.io/getting-started/installation.html ", minReqPodmanVer.String(), v.String())
 		}
 
+		if v.GTE(podmanVerTwo) {
+			return registry.State{
```

Comment:
given that podman might be fixed, how about instead of retruning an error, we just put a warrrning to the user

something like this.
out.WarningT("podman 2 is not supported yet, you might face issues,....")

Reply:
Yeah, that makes more sense to me. Especially considering how <podman 1.7 is handled.

At the time I was just following the desc of the issue.
> we should exit early and advice them use latest v1.9 release